app/javascript/schools/courses/components/curriculum_editor/CurriculumEditor__ContentBlockCreator.re

```diff
@@ -198,34 +213,75 @@ let button = (target, aboveContentBlock, send, addContentBlockCB, blockType) =>
   </label>;
 };
 
+let handleVimeoVideoUpload = (file, vimeo_video) => {
+  Js.log(vimeo_video)
+  let uploadUrl = vimeo_video##uploadUrl
```

Comment:
This should probably be `vimeoVideo##uploadLink`. There is no field `uploadUrl` on the GraphQL response. It's called `uploadLink` instead.